src/main/resources/config/application-dev.yml

```diff
@@ -3,66 +3,87 @@
+# More information on profiles: https://www.jhipster.tech/profiles/
+# More information on configuration properties: https://www.jhipster.tech/common-application-properties/
 # ===================================================================
 
 # ===================================================================
 # Standard Spring Boot properties.
 # Full reference is available at:
 # http://docs.spring.io/spring-boot/docs/current/reference/html/common-application-properties.html
 # ===================================================================
+
+logging:
+    level:
+        ROOT: DEBUG
+        io.github.jhipster: DEBUG
+        com.icthh.xm.uaa: DEBUG
+
 spring:
     profiles:
         active: dev
-        include: swagger
+        include:
+            - swagger
+            # Uncomment to activate TLS for the dev profile
+            #- tls
     devtools:
         restart:
             enabled: true
+            additional-exclude: .h2.server.properties
         livereload:
-            enabled: false # we use gulp + BrowserSync for livereload
+            enabled: false # we use Webpack dev server + BrowserSync for livereload
     jackson:
-        serialization.indent_output: true
+        serialization:
+            indent-output: true
     cloud:
         consul:
             discovery:
                prefer-ip-address: true
             host: localhost
             port: 8500
+        stream:
+            kafka:
+                binder:
+                    brokers: localhost
+                    zk-nodes: localhost
+            bindings:
+                output:
+                    destination: topic-jhipster
     datasource:
         type: com.zaxxer.hikari.HikariDataSource
-        url: jdbc:h2:mem:uaa;DB_CLOSE_DELAY=-1
+        url: jdbc:h2:mem:uaa;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=FALSE
         username: uaa
         password:
+        hikari:
+            auto-commit: false
     h2:
         console:
-            enabled: true
+            enabled: false
     jpa:
         database-platform: io.github.jhipster.domain.util.FixedH2Dialect
         database: H2
-        show-sql: false
+        show-sql: true
         properties:
             hibernate.id.new_generator_mappings: true
+            hibernate.connection.provider_disables_autocommit: true
             hibernate.cache.use_second_level_cache: false
             hibernate.cache.use_query_cache: false
             hibernate.generate_statistics: true
-            #hibernate.cache.region.factory_class: com.hazelcast.hibernate.HazelcastLocalCacheRegionFactory
-            #hibernate.cache.hazelcast.instance_name: uaa
-            hibernate.cache.use_minimal_puts: true
-            #hibernate.cache.hazelcast.use_lite_member: true
+    liquibase:
+        contexts: dev
+        enabled: true
     mail:
-            host:
-            port:
-            username:
-            password:
-            properties:
```

Comment:
Need to check if absence of these properties will not break env variables injected into docker container. 
@ykruchko what do you think?

Reply:
i will check

Reply:
everything ok